style_guides/accessibility_guide.md

```diff
@@ -50,6 +50,39 @@ You should always prefer the `<label for>` solution, since it also adds benefit
 for every user, by making the label clickable, to directly jump to the form
 element (or in case of checkboxes and radio buttons directly check them).
 
+#### How to generate ids?
+
+When labeling element (and for some other accessibility tasks) you will often need
+ids. Ids must be unique within the page (i.e. no duplicate ids in the rendered DOM
+at any time).
```

Comment:
Why have this in parentheses? `Ids must be unique within the page, i.e., no duplicate ids in the rendered DOM
at any time.` Very nit picky, but I think people will read it with more importance when it's not in `( )`.

Reply:
Yeah, I just tend to spread parentheses all over the place (in English and German) :D ... Darn did it again...

Reply:
I do the same 🙈